app/api/migrations/migrations/43-change-system-translation-label/specs/43-change-system-translation-label.spec.js

```diff
@@ -0,0 +1,51 @@
+import testingDB from 'api/utils/testing_db';
+import migration from '../index.js';
+import fixtures from './fixtures.js';
+
+describe('migration change-system-translation-label', () => {
+  beforeEach(async () => {
+    spyOn(process.stdout, 'write');
+    await testingDB.clearAllAndLoad(fixtures);
+  });
+
+  afterAll(async () => {
+    await testingDB.disconnect();
+  });
+
+  it('should have a delta number', () => {
+    expect(migration.delta).toBe(43);
+  });
+
+  it('should update system labels to User Interface', async () => {
+    const expectedLabel = 'User Interface';
+    await migration.up(testingDB.mongodb);
+    const [en, es, pt] = await testingDB.mongodb
+      .collection('translations')
+      .find({})
+      .toArray();
+
+    expect(en.contexts[0].label).toEqual(expectedLabel);
+    expect(es.contexts[0].label).toEqual(expectedLabel);
+    expect(pt.contexts[0].label).toEqual(expectedLabel);
+  });
+
+  it('should not update contexts without system id', async () => {
+    const context = {
+      id: 'somecontext',
+      label: 'System',
+    };
+
+    await testingDB.clearAllAndLoad({
+      translations: [{ contexts: [{ ...context }] }],
+    });
```

Comment:
There are already some other contexts different from the "system" loaded in the fixtures, can you use them instead for the check?

Reply:
done

Reply:
With the last changes the test is not consistent with the description, I think the assert should be something like ```expect(enTranslations.contexts[1].label).toEqual('Other');```

Reply:
fixed